pure-pulsars/src/cmds/user_info.py

```diff
@@ -0,0 +1,82 @@
+"""User info command for the Wiki Guesser game."""
+
+import logging
+from datetime import UTC, datetime
+
+import discord
+import humanize
+from discord import app_commands
+from discord.app_commands.errors import CommandInvokeError
+from discord.errors import NotFound
+
+from database.database_core import DATA, NullUserError
+
+SKIP = ["userid", "last_played"]
+
+
+def main(tree: app_commands.CommandTree) -> None:
+    """Create user-info command."""
+
+    @tree.command(
+        name="user-info",
+        description="Returns your stats",
+    )
+    @app_commands.describe(user="Who are you asking about, leave blank for self?")
+    async def user_info(interaction: discord.Interaction, user: discord.User = None) -> None:
+        """Get game info for a specific user from the scores database.
+
+        Args:
+        ----
+        interaction (discord.Interaction): The interaction object.
+        user (discord.User): The user to get the game info for.
+
+        Notes:
+        -----
+        If no user is provided, the command will default to the user who invoked the command.
+
+        """
+        try:
+            await interaction.response.defer(thinking=True, ephemeral=True)
+            embed = discord.embeds.Embed()
+            try:
+                if user is None:
+                    user = interaction.user
+                user_data = await DATA.get_user(interaction.guild_id, user.id)
+                img = user.display_avatar
+                embed.set_thumbnail(url=img.url)
+                embed.add_field(
+                    name="Name",
+                    value=user_data["name"],
+                    inline=False,
+                )
+                embed.add_field(
+                    name="Score",
+                    value=user_data["score"],
+                    inline=False,
+                )
+                try:
```

Comment:
Could we pre-generate the value variable first, instead of using another nested try/except?

```py
value = (user_data["wins"] / user_data["failure"]) if user_data["failure"] > 0 else user_data["wins"]
```